### ISSUE TITLE
Support for downloading chromedriver at runtime

### ISSUE THREAD

**Issue by HaimBendanan** [giggio/node-chromedriver#256]
Hey, thanks for this very useful repo.
In my use case, I am shipping an app that uses the chrome driver, so I need to handle cases where users have different chrome versions, and where they update chrome once in a while.
I ended up calling the install script every time my app start, to install the appropriate chrome version
Would be great if it was a built in functionality (I had to do some workarounds to make it works)
What do you think about it?

**Comment by giggio** [comment]
Yes, it could be done. We'd only have to add a `download` function to `lib/chromedriver.js`. I'm open to a PR.

**Comment by chbota** [comment]
I've run into a scenario where I think I'd need this functionality. Just a heads up that I'm going to try to get a PR together over the next few days and send it out for feedback 😄

**Comment by r3pwn** [comment]
@chbota I have done the work required for this issue in PR #278, but it is awaiting review :)

**Comment by chbota** [comment]
Even better! Thanks for the heads up 😄

**Comment by bhuber2010** [comment]
@giggio what's the plan on this? I saw the PR went stale. Seemed rather minor of a change really. Being able to call the install outside of the node require to actually wait for it to finish would be huge!

**Comment by giggio** [comment]
@bhuber2010 it went stale because it was a large change and I couldn't review it. If you are able to provide a localized update I'm willing to look into it.

**Comment by niehusst** [comment]
I opened a new PR for this issue that is exactly the same as @r3pwn 's original PR but without the large refactors that made it hard for @giggio to review. 
#332
